docs/core/compatibility/breaking-changes.md

```diff
@@ -7,20 +7,20 @@ ms.date: "08/12/2019"
 
 The following version selector provides a filtered list of breaking changes for .NET Core, ASP.NET Core, and EF Core. The target versions include not only major releases, such as .NET Core 2.2 and .NET Core 3.0, but also preview releases, such as .NET Core 3.0 Preview 7.
 
-You can also view individual issues that detail the breaking changes introduced in .NET Core 3.0 in the following GitHub repositories:
+You can also view individual issues that detail the breaking changes introduced in .NET Core in the following GitHub repositories:
 
-- For .NET Core, the [dotnet/docs](https://github.com/dotnet/docs/issues?q=is%3Aopen+is%3Aissue+label%3Abreaking-change) repository.
+- For .NET Core, the [dotnet/docs](https://github.com/dotnet/docs/issues?q=is%3Aissue+label%3Abreaking-change) repository.
 - For ASP.NET Core, the [aspnet/Announcements](https://github.com/aspnet/Announcements/issues?q=is%3Aissue+is%3Aopen+label%3A%22Breaking+change%22+label%3A3.0.0) repository.
 - For Entity Framework Core, the [aspnet/EntityFrameworkCore](https://github.com/aspnet/EntityFrameworkCore/issues?q=is%3Aopen+is%3Aissue+label%3Abreaking-change) repository.
```

Comment:
For EF Core, I'm not sure if the issues query should filter on is:open or not. Who should I ask about this? @mairaw